annotations/src/main/java/io/quarkiverse/operatorsdk/annotations/RBACCRoleRef.java

```diff
@@ -0,0 +1,16 @@
+package io.quarkiverse.operatorsdk.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ ElementType.TYPE })
+public @interface RBACCRoleRef {
```

Comment:
Wouldn't it make sense for this new annotation to be repeatable, i.e. a Reconciler could be assigned to multiple roles?

Reply:
I think it could have sense

Reply:
done